.github/workflows/api_test.yaml

```
@@ -0,0 +1,22 @@
+jobs:
+  api_test:
+    runs-on: ubuntu-latest
+    name: API Test
+    services:
+      api:
+        image: ghcr.io/alphaspheredotai/vocalizr:latest
+        ports:
+          - 7860:7860
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)